hibernate-core/src/main/java/org/hibernate/dialect/DB2400V7R3Dialect.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.dialect;
+
+import org.hibernate.dialect.identity.DB2IdentityColumnSupport;
+import org.hibernate.dialect.identity.IdentityColumnSupport;
+import org.hibernate.dialect.unique.DefaultUniqueDelegate;
+import org.hibernate.dialect.unique.UniqueDelegate;
+
+/**
+ * An SQL dialect for i. This class provides support for DB2 Universal Database for i V7R1 and
+ * later, also known as DB2/400.
+ *
+ * @author Pierrick Rouxel (pierrickrouxel)
+ */
+public class DB2400V7R3Dialect extends DB2400Dialect {
```

Comment:
I changed the release number because the features used to upgrade the dialect are available in V7R1, but only installing Technology Refresh 11, also available in V7R2 with TR3 but in V7R3 they are shipped natively. I think it's more reliable. For this reason I fixed the documentation and put the dialect as default for V7R3.